### DISCUSSION TITLE
First contribution problems

### DISCUSSION THREAD

**Discussion by BorcilaVasile** [open-sauced/guestbook#404] [Q&A]
I need some help with this problem. I am not sure what I might have missed. 
![image](https://github.com/user-attachments/assets/1fbb4563-39b2-4712-b057-95f44f4a6475)

**Comment by stephmukami** [comment]
I think the repo has automate tasks that our PRs have to go through before they are merged.This message shows that there hasn't been reviews yet so the merge can't happen
